### PR TITLE
draft: feat(import): add clickup import

### PR DESCRIPTION
**Pull request by de-ar**

### CODE REVIEW

**Comment by robcurtis** [comment]
Is this PR getting merged soon? We're interested in moving over from Clickup.

**Comment by de-ar** [comment]
Earlier I could not test the ClickUp importer. Right now, I'm busy at some other work. So no chance of it being merged anytime soon.

**Comment by Chew** [comment]
Is there any updates, or do you intend to make some? I'd love to take over otherwise (will credit you as the starting point)

**Comment by de-ar** [comment]
Hey @Chew!
I do intend to make updates, but my time has become limited. Please do take over and I'm here if you have any questions at all.

File: packages/import/package.json [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
+  "repository": "https://github.com/linear/linear",
   "license": "MIT",
+  "author": "Linear Orbit, Inc",
   "main": "dist/index-cjs.min.js",
   "module": "dist/index-es.min.js",
   "types": "dist/index.d.ts",
   "bin": {
     "linear-import": "bin/linear-import.js"
   },
-  "repository": "https://github.com/linear/linear",
```
Comment (akashchouhan16) [inline packages/import/package.json:13]: Not a relevant code change

File: packages/import/src/importers/clickupCsv/ClickupCsvImporter.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+const mapPriority = (input: ClickupPriority): number => {
+  const priorityMap = {
+    Urgent: 1,
+    High: 2,
+    Medium: 3,
+    Low: 4,
+  };
+  return priorityMap[input] || 0;
```
Comment (akashchouhan16) [inline packages/import/src/importers/clickupCsv/ClickupCsvImporter.ts:158]: Looks good! ⚡

**Review by akashchouhan16 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**